tox.ini

```diff
@@ -77,7 +78,7 @@ commands =
 description = Static type checking
 base = testenv, lint
 env_dir = {work_dir}/typing
-extras = dev, types
+extras = dev, types, focal-dev
```

Comment:
With a clean tox environment, I ran `tox run -e lint-pyright` and it seamlessly installed the focal version of python-apt on my 22.10 system.  Is that OK?

If it is ok, then awesome! It just worked :)

Reply:
It'll link against your system's libapt, so AFAICT it'll be fine until a backwards-incompatible libapt release, and at that point it should break at the stage of compiling python-apt. But especially for the linters here it really shouldn't matter, since it won't actually execute any of the bound code.